### ISSUE TITLE
Add a class to sidebar.html when running in a private window

### ISSUE THREAD

**Issue by 00dani** [mbnuqw/sidebery#419]
I'm using Sidebery with both regular and private windows, and it'd be nice to have it become Sort of Purple when I'm using a private window, like the rest of the browser UI does - that'll make it easier to differentiate the two window types, as well as have a more consistent look across the window.

I've found I can theme the sidebar to be purple using the provided "styles editor" feature, but that change applies to both regular and private windows. I believe it'd be possible to use some custom CSS with nested selectors, similar to the [Dynamic Native Tabs](https://github.com/mbnuqw/sidebery/wiki/Firefox-Styles-Snippets-(via-userChrome.css)#dynamic-native-tabs) configuration suggested in the wiki, to identify private windows and theme them differently? But I've carefully inspected sidebar.html in my browser and found no classes or other differences between private and regular uses of Sidebery that a CSS selector could hook into.

Basically, I'm thinking Sidebery should add a class to its own `div#root`, something like `is-private`, when it detects it's running in a private window. Then it'll be trivial for users to write a quick CSS style to theme the private version of the sidebar differently, whether it's making it a little purple or doing something completely different.
